.github/CODEOWNERS

```diff
@@ -1,247 +1,309 @@
-# Per-directory ownership and automatic assignment for pull requests.
-
-# IMPORTANT NOTES FOR CODEOWNERS:
-#
-# - If you will be unavailable for more than 24 hours, please replace your
-#   ownership with a delegate, file an issue, and add a todo above the owner
-#   line like so:
-#
-#     TODO(#ISSUE_NUMBER): Revert ownership to @USERNAME after YYYY-MM-DD.
-#
-#   (See oppia/#10250 for an example.) Please make sure to restore ownership after
-#   the above date passes.
-
-# Blanket codeowners
-# This is for the case when new files are created in any directories that aren't
-# covered as a whole, since in these cases, codeowners are not recognized for
-# those files when reviewing, even if the PR does add it. Unless new
-# files/folders are created in the following directories, these codeowners would
-# be superseded by the relevant codeowners mentioned elsewhere in this file.
-# (Reference: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners)
-
-#####################################################################################
-#                                 Blanket ownership                                 #
-#####################################################################################
+################################################################################
+#                        IMPORTANT NOTES FOR CODEOWNERS:                       #
+#                                                                              #
+# - If you will be unavailable for more than 24 hours, please replace your     #
```

Comment:
Can we use codeowner teams instead for this first paragraph? I don't think we should be doing issue filing etc. here, or editing the codeowner file -- that is much too clunky IMO. The procedure should be something like this instead:

(a) make sure there are other folks on your codeowner team
(b) send a note to them and make sure they can cover, and tell them when you'll be back; include Ben/Sean/Adhiambo on that note
(c) update your status to "busy" to make sure you don't get reviews
(d) when you're back, change your status back so that you get new reviews

Reply:
Ah, that's what the new comment was meant to suggest (since I removed the previous instructions that explained adding a TODO). These steps are much more actionable though, so I'll switch it over to these--thanks!

Reply:
Sg, though just to confirm, the updated comment in the PR still has "please replace your ownership with a delegate, file an issue" in it. There's no need to file an issue, right?